src/webview/MessageList.js

```diff
@@ -75,6 +75,7 @@ export type BackgroundData = $ReadOnly<{
   mute: MuteState,
   ownEmail: string,
   ownUserId: number,
+  ownIsAdmin: boolean,
```

Comment:
`isAdmin` will do. (The `own` affix here is of questionable utility anyway, IMO, even where it's grammatical.)

Also, please make sure the ordering of properties is consistent here and below.

Reply:
> (The `own` affix here is of questionable utility anyway, IMO, even where it's grammatical.)

I think it's important for "email" and "user id" and "user". There are lots and lots of places we have a variable (or parameter or property) named `email` or `userId` or `user`, and they normally mean "(the email or ID of) the user this widget, or this scrap of data, is about". When it's instead "the user that is looking at the app", that has a very different feel and it's good to make the contrast explicit.

I agree it's not helpful for "is admin", and probably most other possible names.